### PR TITLE
Fixes and Improved some more caching issues

### PR DESCRIPTION
**Pull request by ManiruzzamanAkash**
[fix] Fixed vendor related caches.

### CODE REVIEW

File: includes/Vendor/VendorCache.php [hunk 1]
```diff
@@ -17,11 +17,24 @@
...
+    }
+
+    /**
+     * Clear vendor cache group.
+     *
+     * @since 3.3.2
+     *
+     * @return void
+     */
+    public function delete( ) {
```
Comment (nurul-umbhiya) [inline includes/Vendor/VendorCache.php:36]: make this method static, this way whole codebase will be consistant

Comment (ManiruzzamanAkash) [reply includes/Vendor/VendorCache.php:36]: ok bhai.

**Comment by shanjidah** [comment]
Issue No 2: Product Review is not working properly. Approve or other buttons are not working properly. The approved list is not showing.

Here is the screenshot:
Screen Shot 2021-11-29 at 11 48 29 AM

**Comment by shanjidah** [comment]
Issue No3: The vendor can not view order details from the manage booking tab.

**Here is the screenshot:**
Screen Shot 2021-11-29 at 12 54 57 PM

**Comment by shanjidah** [comment]
**Issue No5:** After activating Redis caching, the report Abuse menu is continuously loaded.
Screen Shot 2021-11-30 at 9 58 06 AM

**Comment by shanjidah** [comment]
**Issue No:6**  Support ticket listing shows wrong numbers. Open Support tickets tabs show closed ticket status to open.

**Here is the screenshot:**
Screen Shot 2021-11-30 at 11 50 41 AM
Screen Shot 2021-11-30 at 11 53 43 AM

**Review by nurul-umbhiya — APPROVED** [review]

**Pull request merged**